src/plot/plotStore.ts

```diff
@@ -7,6 +7,15 @@ export enum PlotAxes {
   reciprocal = "nm^-1",
 }
 
+// default colours used in app
+const lilac = { r: 144, g: 19, b: 254, a: 0.4 };
+const black = { r: 0, g: 0, b: 0, a: 1 };
+const blackOpaque = { r: 0, g: 0, b: 0, a: 0.2 };
+const turquoise = { r: 80, g: 227, b: 194, a: 0.4 };
+const mustard = { r: 245, g: 166, b: 35, a: 1 };
+const green = { r: 65, g: 117, b: 5, a: 1 };
+const red = { r: 208, g: 2, b: 27, a: 1 };
+
```

Comment:
good extraction for now

the detectorColor property of PlotConfig should be considered to change into string
react-color is not well-maintained - last update 4 years ago
https://www.npmjs.com/package/react-color

using raw hex values with an extension like this (https://marketplace.visualstudio.com/items?itemName=kamikillerto.vscode-colorize) should also provide also excellent developer experience

regarding the input itself it is not necessary to use a react library, a raw html element can be considered

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color

still this is beyond the scope of this PR, I'd suggest making a new issue from this comment and decide what to do later